doc/whats-new.rst

```diff
@@ -196,7 +209,6 @@ Documentation
 - Add :py:meth:`Dataset.polyfit` and :py:meth:`DataArray.polyfit` under "See also" in
   the docstrings of :py:meth:`Dataset.polyfit` and :py:meth:`DataArray.polyfit`
   (:issue:`5016`, :pull:`5020`). By `Aaron Spring <https://github.com/aaronspring>`_.
```

Comment:
could you go through the entries for 0.18.0 and put the names on their own lines? For this one, it would be
```suggestion
  (:issue:`5016`, :pull:`5020`).
  By `Aaron Spring <https://github.com/aaronspring>`_.
```